app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser_dependencies.ts

```diff
@@ -1,6 +1,15 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {intlShape} from 'react-intl';
+import {Alert} from 'react-native';
+
+import type {AutocompleteSuggestion} from '@mm-redux/types/integrations';
```

Comment:
@enahum  should `import type` be a separate block at the end of all other imports block?